lib/workers/repository/config-migration/branch/index.ts

```diff
@@ -18,10 +22,44 @@ export async function checkConfigMigrationBranch(
     return null;
   }
   const configMigrationBranch = getMigrationBranchName(config);
-  if (await migrationPrExists(configMigrationBranch)) {
+  const commitMessageFactory = new ConfigMigrationCommitMessageFactory(
+    config,
+    migratedConfigData.filename
+  );
+  const prTitle = commitMessageFactory.create().toString();
+  const closedPrConfig: FindPRConfig = {
+    branchName: configMigrationBranch,
+    prTitle,
+    state: PrState.Closed,
+  };
+  const branchConfig: BranchConfig = {
+    prTitle,
+    manager: '',
+    upgrades: [],
+    branchName: configMigrationBranch,
+    userStrings: {
+      ...(config.userStrings as Record<string, string>),
+      ignoreOther: '',
+    },
+    suppressNotifications: config.suppressNotifications,
+  };
+
+  const branchPr = await migrationPrExists(configMigrationBranch); // handles open/autoClosed PRs
+  const closedPr = branchPr ? undefined : await platform.findPr(closedPrConfig); // handles closed PR
+
+  // found closed migration PR
+  if (closedPr) {
+    logger.debug(
+      { prTitle: closedPr.title },
+      'Closed PR already exists. Skipping branch.'
+    );
+    await handlepr(branchConfig, closedPr);
```

Comment:
I wonder if this will print an incorrect/misleading message, e.g. "Renovate will ignore this version". I'm OK with leaving this out.

Also, we should probably now document that if you close the migration PR then it will be ignored and not reopened/recreated.

Reply:
> I wonder if this will print an incorrect/misleading message, e.g. "Renovate will ignore this version". I'm OK with leaving this out.

it shouldnt really, as it is dependent on `updateType` which we dont have in migration PRs. other than that i have hard coded `userStrings` to be basically only `ignoreTopic` and its the only field configurable (default = 'Renovate Ignore Notification'), but we can hardcode this as well